requirements.txt

```diff
@@ -1,5 +1,5 @@
 rucio-clients>=1.26.1
 pika==1.1.0
-servicex-did-finder-lib>=1.1a5
+servicex-did-finder-lib==1.1a7
```

Comment:
Let's get the did-finder-lib published to pypi and get a real version before merging this with version 1.2